exec/vector/pom.xml

```diff
@@ -25,7 +25,7 @@
     <groupId>org.apache.drill.exec</groupId>
     <version>2.0.0-SNAPSHOT</version>
   </parent>
-  <artifactId>vector</artifactId>
+  <artifactId>drill-vector</artifactId>
```

Comment:
@luocooong it was seeing the effort already made in [DRILL-7847](https://issues.apache.org/jira/browse/DRILL-7847) that caused me to try to complete the job. I do like completeness and consistency but the name of the vector module also does not keep me awake at night so I won't argue much here. Do you recommend that I revert this part of the PR?

Reply:
Good point.
It is recommended to update the name of the module instead of the ID, and consistency will do better. Because we also need to treat this pull request that allows backport.

Reply:
@luocooong it is only the artifactId that is inconsistent here. The name of `Drill : Exec : Vectors` already looks good to me. So I'll revert this.